src/mavlink-router/mainloop.cpp

```diff
@@ -261,7 +264,11 @@ void Mainloop::loop()
     add_timeout(LOG_AGGREGATE_INTERVAL_SEC * MSEC_PER_SEC,
                 std::bind(&Mainloop::_log_aggregate_timeout, this, std::placeholders::_1), this);
 
-    while (!should_exit) {
+     usec_t timestamp_stop_requested = 0;
+
+     bool run_loop= true;
```

Comment:
There's something wrong here with the indentation

Reply:
@bkueng The script did not catch that so I fixed it manually.